recipes/circos/meta.yaml

```diff
@@ -0,0 +1,72 @@
+package:
```

Comment:
As in the autograd recipe, we would rather use the `pypi.io` url scheme + jinja tags.